compute_sdk/globus_compute_sdk/sdk/client.py

```diff
@@ -591,7 +591,8 @@ def register_function(
         container_uuid : str
             Container UUID from registration with Globus Compute
         description : str
-            Description of the file
+            Description of the function. If this is None, and the function has a
+            docstring, that docstring is uploaded as the function's description.
```

Comment:
So ... this takes priority if the function has a docstring _and_ the user specifies this value?